ocs_ci/ocs/ui/page_objects/premigration_fusion_access_ui.py

```
@@ -0,0 +1,218 @@
+        # each row has its own <input id="node-<uuid>" class="pf-v6-c-check__input">.
+        # PF v6 checkboxes are visually overlaid by a <label>, so a plain click on
+        # the <input> is intercepted.  We use JS .click() to bypass that overlay.
+        logger.info("Step 3: Select all nodes — clicking each unchecked row checkbox")
+        node_checkboxes = self.get_elements(
+            FUSION_ACCESS_STORAGE_CLUSTER_LOCATORS["node_row_checkboxes"]
+        )
+        assert node_checkboxes, (
+            "No node checkboxes found on the 'Create storage cluster' page. "
+            "At least one worker node must be present."
+        )
+        logger.info(f"Found {len(node_checkboxes)} node row(s) in the table")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)